doc/contribute/guidelines.rst

```diff
@@ -532,9 +532,17 @@ results you have to create an account yourself.  From the Zephyr project page,
 you may select "Add me to project" to be added to the project. New members must
 be approved by an admin.
 
-Coverity scans the Zephyr codebase weekly. GitHub issues are automatically
-created for any problems found and assigned to the maintainers of the affected
-areas.
+Static analysis in the Zephyr codebase happens bi-weekly. GitHub issues are
+automatically created for any issues detected by Coverity. These issues will
+have the same (or equivalent) priority initially defined by the tool.
+
+To ensure accountability and efficient issue resolution, they are assigned to
+the respective code owner who is responsible for the affected code.
+
+A dedicated team comprising members with expertise in static analysis, code
+quality, and software security is responsible to ensure the effectiveness of
+the static analysis process and verify that identified issues are properly
```

Comment:
nit: grammar.
I suggest either:
```suggestion
quality, and software security ensure the effectiveness of
the static analysis process and verify that identified issues are properly
```
or
```suggestion
quality, and software security is responsible for ensuring the effectiveness of
the static analysis process and verifying that identified issues are properly
```

Reply:
I like option 1.

Reply:
Option 1 would be "ensures", not "ensure"